cfme/services/myservice.py

```diff
@@ -202,20 +202,27 @@ def download_file(self, extension):
         navigate_to(self, 'All')
         down_btn("Download as " + extension)
 
+    @property
+    def label_all_services(self):
```

Comment:
Let's change this to `all_services_label` instead, please. Method names that start with a verb suggest action so it is not ideal for a property.